pages/reference.mdx

```diff
@@ -40,49 +40,49 @@ In API v8, we've improved error formatting in form error responses. The response
 
 ```json
 {
-  "code": 50035,
-  "errors": {
-    "activities": {
-      "0": {
-        "platform": {
-          "_errors": [
-            {
-              "code": "BASE_TYPE_CHOICES",
-              "message": "Value must be one of ('desktop', 'android', 'ios')."
-            }
-          ]
-        },
-        "type": {
-          "_errors": [
-            {
-              "code": "BASE_TYPE_CHOICES",
-              "message": "Value must be one of (0, 1, 2, 3, 4, 5)."
-            }
-          ]
-        }
-      }
-    }
-  },
-  "message": "Invalid Form Body"
+	"code": 50035,
+	"errors": {
+		"activities": {
```

Comment:
How do tabs look in the docs?

Reply:
Interesting, I thought I disabled prettier for mdx files....cause IIRC tabs don't look great (although maybe we should make them look great?)

Reply:
Thanks for pointing out that I missed running prettier on the rest of the mdx files 😄

Reply:
Oh...disabled eslint, but prettier still runs (and should), just gonna override tabs for mdx files

Reply:
It should still use tabs imo, but change the tab-size in css to 2 (or let users configure it)

Reply:
yeah, I did a simple change for now, but ideally it'd be tabs with a tab size selector somewhere in the longrun